build.gradle

```diff
@@ -25,9 +30,10 @@ def todaysDate = LocalDate.now().format('yyyy-MM-dd')
 version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
 
 jar {
-    baseName = rootProject.name
+    archiveBaseName.set(rootProject.name)
+
     manifest {
-        attributes 'Implementation-Title': rootProject.name, 'Implementation-Version': version
+        attributes 'Implementation-Title': rootProject.name, 'Implementation-Version': archiveVersion.get()
```

Comment:
what's this doing?

Reply:
on second thoughts that whole `jar` section can be removed - it's certainly not adding anything to META-INF/MANIFEST.MF

Reply:
Repacing another deprecated getter method